src/pages/journal/journal.ts

```diff
@@ -1,25 +1,44 @@
-import { Component, OnInit, OnDestroy, ViewChild, ViewContainerRef, ComponentFactoryResolver } from '@angular/core';
-import { IonicPage, LoadingController, NavController, NavParams, Platform, ToastController, Loading, Toast, Refresher } from 'ionic-angular';
-import { Store, select } from '@ngrx/store';
+import { Component, ComponentFactoryResolver, OnDestroy, OnInit, ViewChild, ViewContainerRef } from '@angular/core';
+import {
+  IonicPage,
```

Comment:
Not sure if this a good idea. What is wrong with having it in one line?

Reply:
It _should_ be beyond the line length limit of the linter.